src/building/new-target.md

```diff
@@ -8,28 +8,22 @@ relevant to your desired goal.
 
 For very new targets, you may need to use a different fork of LLVM
 than what is currently shipped with Rust. In that case, navigate to
-the `src/llvm_project` git submodule (you might need to run `x.py
+the `src/llvm-project` git submodule (you might need to run `x.py
 check` at least once so the submodule is updated), check out the
 appropriate commit for your fork, then commit that new submodule
 reference in the main Rust repository.
 
 An example would be:
 
 ```
-cd src/llvm_project
+cd src/llvm-project
 git remote add my-target-llvm some-llvm-repository
 git checkout my-target-llvm/my-branch
 cd ..
 git add llvm_target
 git commit -m 'Use my custom LLVM'
 ```
 
-If you have a local LLVM checkout that is already built, you *may* be
-able to configure Rust to treat your build as the [system
-LLVM][sysllvm] to avoid redundant builds.
-
-[sysllvm]: ./suggested.md#skipping-llvm-build
```

Comment:
It seems unfortunate to have to delete this information, but the key information was already deleted.

Reply:
I think rather than deleting this, we should move the system-llvm section here. I hadn't thought of someone who had a custom LLVM fork already built, this would be useful for them.

Reply:
What is the "system-llvm" section? Do you mean the part I added the begins with

> Note that by default, `x.py setup` will use CI-built LLVM

?

Reply:
> What is the "system-llvm" section?

I mean the section I removed in https://github.com/rust-lang/rustc-dev-guide/pull/1030.

Reply:
Okay, does this look good: 2d42cf7?

Reply:
Looks great, thanks!